Source/Scene/Vector3DTileContent.js

```diff
@@ -720,6 +721,9 @@ Vector3DTileContent.prototype.update = function (tileset, frameState) {
   }
   if (defined(this._batchTable) && ready) {
     this._batchTable.update(tileset, frameState);
+    if (!defined(this._features)) {
```

Comment:
:+1: So does it make sense to still call `createFeatures` in `getFeature` or `applyStyle`?

Reply:
Also please update `CHANGES.md` with the fix.

Reply:
> 👍 So does it make sense to still call createFeatures in getFeature or applyStyle?

The issue in the Sandcastle provided offline was caused because of this. A user may set the style before the update command and so, if the features are not created in the `update`, they need to be created in `applyStyle`. In both places, we do check if they're already created to avoid re-creating them.